fvm/evm/stdlib/contract.cdc

```diff
@@ -69,9 +69,14 @@ contract EVM {
     resource BridgedAccount {
 
         access(self)
-        let addressBytes: [UInt8; 20]
+        var addressBytes: [UInt8; 20]
 
-        init(addressBytes: [UInt8; 20]) {
+        init() {
+            self.addressBytes = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        }
+
+        access(contract)
+        fun setAddress(addressBytes: [UInt8; 20]) {
```

Comment:
This change seems a bit odd: Why do bridged accounts now initially have an empty address? Why/how is t possible that the address may change at some point?

Is it because the resource's UUID is now needed to determine the address?

Maybe leave some comments in the contract here explaining this (why address is initially empty, when the address can change, etc.)

Reply:
updated the doc and also added a check to make sure initAddress can only be called once.

Reply:
Nice! 👌